### PR TITLE
aragon docs

### PR DESCRIPTION
**Pull request by J-Thompson12**
Closes: #XXX

## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+  title: 'Aragon-Chain Documentation',
+  locales: {
+    '/': {
+      lang: 'en-US'
+    },
+  },
+  base: process.env.VUEPRESS_BASE || '/',
+  themeConfig: {
+    repo: 'ChainSafe/aragon-chain',
+    docsRepo: 'ChainSafe/aragon-chain',
```
Comment (J-Thompson12) [inline docs/.vuepress/config.js:12]: I was never able to get npm to run on my machine. I dont know if there are plans to create an aragon chain repo or if this should be the ethermint repo and the aragon branch

File: docs/.vuepress/config.js [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+              path: '/modules'
+            }
+          ]
+        },
+        {
+          title: 'Resources',
+          children: [
+            {
+              title: 'Aragon-Chain API Reference',
+              path: 'https://godoc.org/github.com/cosmos/aragon-chain'
```
Comment (J-Thompson12) [inline docs/.vuepress/config.js:74]: same with this one.

**Comment by fedekunze** [comment]
shouldn't this be done on the aragon chain repo? https://github.com/aragon/aragon-chain

**Comment by J-Thompson12** [comment]
Ah yes thank you. I couldnt find it in chainsafe so that helps

**Pull request closed without merging**